examples/complex/Main.lhs

```diff
@@ -105,13 +131,58 @@ observeIO trace = do
   where
     loop tr = do
         threadDelay 1000000  -- 1 second
-        bracketObserveIO trace "observeIO" $ do
+        bracketObserveIO tr "observeIO" $ do
```

Comment:
right! we probably do not need the extra parameter if we never change the trace.